Containerfile

```diff
@@ -149,6 +149,8 @@ CMD ["bundle", "exec", "rails", "server", "-b", "0.0.0.0", "-p", "3000"]
 ## Development
 FROM base AS development
 
+ENV TMPDIR=/tmp
```

Comment:
Oddly, without this, `bin/setup` will struggle in Codespace stating that RUby could not time tmp directory.